compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -119,8 +119,9 @@ setPackageDbs paths dflags =
 setPackageImports :: Bool -> [(UnitId, ModRenaming)] -> DynFlags -> DynFlags
 setPackageImports hideAllPkgs pkgImports dflags = dflags {
     packageFlags = packageFlags dflags ++
-        [ExposePackage ("-package " <> unitIdString pkgName) (UnitIdArg pkgName) renaming
-        -- The first string is only used in error messages.
+        [ExposePackage ("-package-id " <> unitIdString pkgName) (UnitIdArg pkgName) renaming
```

Comment:
Is this a user-facing change which needs documenting and/or added to the change log?

Reply:
No, as mentioned in the comment this is only something used in error messages and we don’t really expect users to use the `-package` flag at all. That said, after thinking about it a bit more I’ve changed it to `--package` which makes more sense since that’s the flag name accepted by `damlc` (`-package-id` is what GHC calls the corresponding flag).